app/assets/stylesheets/components/_header.scss

```diff
@@ -0,0 +1,17 @@
+.sc-TakeoverHeader {
+
+  &-content {
+    @include display(flex);
+    @include justify-content(space-between);
+  }
+
+  &--multipleCta {
+    @include display(flex);
+
+    a {
```

Comment:
These rules are not specific to the TakeoverHeader component. If we need special link rules, let's use a class selector in some app-specific link component / utility.

Taken together, that means we should be able to ditch this `multipleCta` component variant altogethe.